rewrite-gradle/src/main/java/org/openrewrite/gradle/AddDependencyVisitor.java

```diff
@@ -290,22 +290,10 @@ public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, Execu
                             }
                         } else {
                             Space originalPrefix = addDependencyInvocation.getPrefix();
-                            if (currentStatement instanceof J.VariableDeclarations) {
-                                J.VariableDeclarations variableDeclarations = (J.VariableDeclarations) currentStatement;
-                                if (variableDeclarations.getTypeExpression() != null) {
-                                    addDependencyInvocation = addDependencyInvocation.withPrefix(variableDeclarations.getTypeExpression().getPrefix());
-                                }
-                            } else {
-                                addDependencyInvocation = addDependencyInvocation.withPrefix(currentStatement.getPrefix());
-                            }
+                            addDependencyInvocation = addDependencyInvocation.withPrefix(currentStatement.getPrefix());
 
                             if (addDependencyInvocation.getSimpleName().equals(beforeDependency.getSimpleName())) {
-                                if (currentStatement instanceof J.VariableDeclarations) {
-                                    J.VariableDeclarations variableDeclarations = (J.VariableDeclarations) currentStatement;
-                                    if (variableDeclarations.getTypeExpression() != null && !variableDeclarations.getTypeExpression().getPrefix().equals(originalPrefix)) {
-                                        statements.set(i, variableDeclarations.withTypeExpression(variableDeclarations.getTypeExpression().withPrefix(originalPrefix)));
-                                    }
-                                } else if (!currentStatement.getPrefix().equals(originalPrefix)) {
+                                if (!currentStatement.getPrefix().equals(originalPrefix)) {
```

Comment:
It is very unclear to me what these changes here have to do with the parsing issue. Is a comment required here?

Reply:
Due to the way `DeclarationExpression`s were parsed into `J.VariableDeclarations` we needed a specific check here. After my changes the parsing is more in-line with other types and so we no longer need that exceptional case here!